### PR TITLE
feat(Slider): add Slider element

### LINKED ISSUE #75: Add Slider element
Add a Slider - a progress control component, similar to the DOM's `input[type="range"]`

### PR DESCRIPTION
**Pull request by THoj13**
- moved `FadeShader` to `textures/` , exported from root
- added `Slider` element

### CODE REVIEW

File: textures/index.js [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+      ctx.closePath();
+      ctx.stroke();
+      ctx.fill();
+
+      cb(null, { source: canvas, w, h, direction });
+    };
+  }
+}
+
+export { FadeShader, Circle, Arrow };
```
Comment (anthony9187) [inline textures/index.js]: can we split these out to separate files and export them from the textures/index.js?

Comment (THoj13) [reply textures/index.js]: Can do!

**Review by anthony9187 — APPROVED** [review]

**Review by erautenberg — CHANGES_REQUESTED** [review]
This is on a white background for me. Is that true for anyone else? Shouldn't it be a white slider on black background?

**Comment by anthony9187** [comment]
> This is on a white background for me. Is that true for anyone else? Shouldn't it be a white slider on black background?

I'm seeing this as well, can we update this so the story matches the others?

**Comment by THoj13** [comment]
> > This is on a white background for me. Is that true for anyone else? Shouldn't it be a white slider on black background?
> 
> I'm seeing this as well, can we update this so the story matches the others?

Updated to use dark background

**Review by erautenberg — APPROVED** [review]

**Pull request merged**